src/main/java/com/github/lunatrius/schematica/client/printer/SchematicPrinter.java

```diff
@@ -194,7 +194,7 @@ private boolean placeBlock(final WorldClient world, final EntityPlayerSP player,
             return false;
         }
 
-        if (ConfigurationHandler.destroyBlocks && !world.isAirBlock(realPos) && this.minecraft.playerController.isInCreativeMode()) {
+        if (ConfigurationHandler.destroyBlocks && !world.isAirBlock(realPos) && this.minecraft.playerController.isInCreativeMode() && PlacementRegistry.INSTANCE.getPlacementData(blockState, itemStack).getExtraClicks(blockState) == 0) {
```

Comment:
What exactly is this change supposed to do? Doesn't really have anything to do with the original issue...